htsget-actix/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "htsget-actix"
 version = "0.4.1"
-rust-version = "1.64"
+rust-version = "1.65"
```

Comment:
Did you run `cargo-msrv` here? Wondering if we can bump the other crates accordingly.

Reply:
Yes. `cargo msrv` returned different msrvs for different crates. We could make them all the same? Although I think it's okay as is, because some htsget-rs crates could be used independently of the others.

Reply:
Nah, leave it as is, most folks go latest rust versions anyway.